### ISSUE TITLE
Ignoring Tab/Esc/Enter-Key

### ISSUE THREAD

**Issue by RaeuberHotzenplotz** [jacobslusser/ScintillaNET#205]
When having the Scintilla component embedded in a (modal) form, it would be nice to have an option that makes the editor ignore special keys like Tab/Escape/Enter and let them bubble up to its parent control. That way you don't have to handle e.g. the KeyPress event yourself and call Control.SelectNextControl for the Tab key or trigger the form's cancel button on Escape.

Perhaps there's already an elegant way that i don't see.

**Comment by jacobslusser** [comment]
See issue #101 for a possible answer.

**Comment by RaeuberHotzenplotz** [comment]
Yes, overriding ProcessCmdKey is also an option. I was just asking for an out-of-the-box solution like a method scintilla.IgnoreKey(Keys.Tab) that would simplify the problem.
